docs/entity-cheat-sheet.rst

```diff
@@ -0,0 +1,333 @@
+.. spelling:word-list::
+    hh
+    num
+
+.. comment
+   add an og image like this :og:image: https://docs.getodk.org/_static/img/tutorial-first-form.png
+
+Entity Cheat Sheet
+=================================
+
+This page provides a quick reference for how to design forms in :doc:`XLSForm </tutorial-first-form/>` for ODK that use, create, and update Entities.
```

Comment:
Maybe Entities can link to the intro doc?

Reply:
sounds good! I'm adding a link back to the entity cheat sheet on that page, too. 
> For a quick reference, check out the [Entity Cheat Sheet](http://0.0.0.0:8000/entity-cheat-sheet/).